src/features/vote/proposals/list/ProposalInterestVote.tsx

```diff
@@ -0,0 +1,95 @@
+import { Label } from '@radix-ui/react-label';
+import { MinusIcon, ThumbsDownIcon, ThumbsUpIcon } from 'lucide-react';
+import {
+  RadioGroup,
+  RadioGroupItem,
+} from '../../../../components/ui/radio-group.tsx';
+import { cn } from '../../../../utils.ts';
+import type { Proposal, Vote } from '../types.ts';
+
+const voteOptions = [
+  {
+    label: 'Strongly Disinterested',
+    value: '-2',
+    id: 'vote-strongly-disinterested',
+  },
+  {
+    label: 'Slightly Disinterested',
+    value: '-1',
+    id: 'vote-slightly-disinterested',
+  },
+  { label: 'Neutral', value: '0', id: 'vote-neutral' },
+  { label: 'Slightly Interested', value: '1', id: 'vote-slightly-interested' },
+  { label: 'Strongly Interested', value: '2', id: 'vote-strongly-interested' },
+];
+
+type ProposalInterestVoteProps = {
+  proposal: Proposal;
+  disabled?: boolean;
+  onVote: (vote: Vote) => void;
+  value: Vote;
+};
+
+export default function ProposalInterestVote({
+  proposal,
+  disabled = false,
+  onVote,
+  value,
+}: ProposalInterestVoteProps) {
+  return (
+    <RadioGroup
+      aria-label="Vote"
+      className="flex flex-col md:flex-row md:items-center gap-2"
+      value={value}
+      onValueChange={onVote}
+      disabled={disabled}
+    >
+      {voteOptions.map((option) => {
+        const optionValue = parseInt(option.value, 10);
+
+        return (
+          <Label
+            key={`${proposal.id}-${option.id}`}
+            className={cn('flex items-center gap-2', {
```

Comment:
POLISH: The radio buttons look pretty wonky to me. That's mostly because we have an odd number of off-balance items, but I wonder moving the labels below the radio buttons might help it look more even?
SUGGESTION:
```suggestion
            className={cn('flex w-[20%] flex-col text-center items-center gap-2', {
```

Reply:
You're right, I agree with that. I've made an update to reflect this.